modules/ROOT/pages/managing-users.adoc

```diff
@@ -91,6 +91,68 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
+
+If you are migrating from the default Anypoint Platform SSO certificate to a new certificate, you must update the Assertion Consumer Service (ACS) URL in your IdP. Only the new ACS URL supports the keys generated by the key rotation feature. If your ACS URL already follows the pattern of `.../accounts/login/:org-domain/providers/:providerId/receive-id`, you do not need to change the ACS URL.
+
+=== Generate a Key
+
+Anypoint Platform can create a key for you to use with your IdP.
+
+. In Anypoint Platform, sign in as a user with the Organization Administrator permission.
+. In *Management Center*, click *Access Management*.
+. In the navigation menu on the left, click *Identity Providers*.
+. Next to a SAML 2.0 IdP, click *Edit*.
+. Click the *Anypoint Keys* tab.
+. Click *+ New key*, and select *Generate*. +
+A newly generated key appears in the list of keys.
+
+=== Upload a Key
+
+You can upload an existing public and private key pair.
+
+. In Anypoint Platform, sign in as a user with the Organization Administrator permission.
+. In *Management Center*, click *Access Management*.
+. In the navigation menu on the left, click *Identity Providers*.
+. Next to a SAML 2.0 IdP, click *Edit*.
+. Click the *Anypoint Keys* tab.
+. Click *+ New key*, and select *Upload*.
+. In the *Private key* field, paste your private key.
+. In the *Public key* field, paste your certificate.
+. Click *Upload*. +
+The key you uploaded appears in the list of keys.
+
+=== Change the Primary Key
+
+Because you can store up to 3 keys, you must designate a _primary key_ to be used for signing requests from Anypoint Platform.
+Before reassigning the primary key, ensure your IdP can access the key that you intend to use.
+
+. In Anypoint Platform, sign in as a user with the Organization Administrator permission.
+. In *Management Center*, click *Access Management*.
+. In the navigation menu on the left, click *Identity Providers*.
+. Next to a SAML 2.0 IdP, click *Edit*.
+. Click the *Anypoint Keys* tab.
+. Next to the key that you want to make primary, click *...* and select *Use as primary key...*.
+. Click *Apply*.
+
+=== Revoke a Key
+
+As a best practice, revoke a key that you have already used as the primary key.
+Before you revoke a key, ensure your IdP is not using the key.
+Note that when you revoke a public key, Anypoint Platform also revokes the corresponding private key. For security purposes, you cannot retrieve a key that has been revoked.
+
+. In Anypoint Platform, sign in as a user with the Organization Administrator permission.
+. In *Management Center*, click *Access Management*.
+. In the navigation menu on the left, click *Identity Providers*.
+. Next to a SAML 2.0 IdP, click *Edit*.
+. Click the *Anypoint Keys* tab.
+. Next to the key that you want to revoke, click *...* and select *Revoke*.
+. Click *Revoke*.
+
 == Import SAML Metadata
 
```

Comment:
Add text to tie this to the focus of this topic set, Key Rotation.
After reconstructing the topic to actually FOCUS on key rotation....

Reply:
Unrelated content.